### DISCUSSION TITLE
Связь

### DISCUSSION THREAD

**Discussion by Krukrukruzhka** [calculation-methods/M8O-303B-21#5] [Q&A]
Можно ли задавать вопросы где-то помимо раздела дискуссий? Например, тг/вк/whatsapp

**Comment by pivovarov-mai** [comment]
Если удобнее, то конечно можно. Но предупреждаю, что ответы не будут молниеносными.

**Reply by pivovarov-mai** [reply]
Еще можно писать на почту `pivovarov@mai.ru`
